### ISSUE TITLE
Capital 'o' and zero characters are impossible to tell which is which.

### ISSUE THREAD

**Issue by LeviticusNelson** [vercel/geist-font#23]
Is there a design choice to why these two are essentially the same design? As a developer using geist mono, I feel like there should be a different design for the zero char to tell the difference without a second look.

I provided what the two characters look like side-to-side:

<img width="2672" alt="Screenshot 2023-10-28 at 12 40 59 PM" src="https://github.com/vercel/geist-font/assets/54561718/27d26446-d247-4263-9454-573124539e34">

**Comment by christianlegge** [comment]
Thanks for creating this issue; I was considering trying out this font and now I don't have to bother.

**Comment by adamdotjs** [comment]
It doesn't seem to be mentioned anywhere, but there is a stylistic set for the zero character (and others). If your IDE supports them, you can enable it with the `ss09` flag.

![image](https://github.com/vercel/geist-font/assets/68607953/8a6faab9-b52c-4fe4-9098-b4a2b7a13250)

**Comment by skrewde** [comment]
This is a pretty serious issue, hope it gets addressed soon.

**Comment by Emonadeo** [comment]
It might also be a good idea to use the `"zero"` feature tag instead of the arbitrary stylistic set `"ss09"`.

This would also enable it to be activated in CSS like this:
```css
font-variant-numeric: slashed-zero;
```
instead of this:
```css
font-feature-settings: "ss09";
```

**Comment by daiyam** [comment]
I think the `slashed-zero` should be the default zero. In the current state, the zero and O are too much similar.

**Comment by JohnPhamous** [comment]
Fixed in #46 and available in the `1.0.1` release on [GitHub](https://github.com/vercel/geist-font/releases/tag/1.0.1) and on [npm](https://www.npmjs.com/package/geist).
